### ISSUE TITLE
Add PIRB

### ISSUE THREAD

**Issue by Muennighoff** [embeddings-benchmark/mteb#1265]
Add datasets not yet in MTEB from https://github.com/sdadas/pirb! Would be cool to support this as a benchmark too.
cc @rafalposwiata maybe you know which ones are still missing here? Amazing work on the benchmark btw!

**Comment by rafalposwiata** [comment]
Most of the PIRB datasets are not included in the MTEB. My comment relative to each group of datasets is as follows:

- PolEval2022 - datasets are part of the larger and newer PolQA (https://huggingface.co/datasets/ipipan/polqa) dataset, which can be added to MTEB.
- Web Datasets - closed access, we will not make them available to the public.
- BEIR-PL - already in MTEB.
- MaupQA - for some of these datasets, the results of even good models are very low, which may suggest that there is something wrong with them. This requires additional review.
- Other - MFAQ (multilingual dataset, I don't know if some version of it is not already in MTEB), GPT-exams (can be added).

In summary, I suggest adding PolQA, MFAQ (if it is not in MTEB), and GPT-exams first. 

In addition, I think that adding many retrieval-type datasets may somewhat disrupt the idea of a comprehensive benchmark. Expanding by a few will be enough.

**Comment by KennethEnevoldsen** [comment]
> Web Datasets - closed access, we will not make them available to the public.

@rafalposwiata how do you support closed-access datasets? Simply running the models yourself? If we wish we could def. support this style as well if that would be something of interest? We will need so distinction to make sure that users are aware that it is a private dataset (I assume it is on HF or similar and accessed using a token?)

**Comment by rafalposwiata** [comment]
@KennethEnevoldsen Yes, we run the models ourselves. We provide web datasets upon request by passing on the download link. It isn't hosted on HF. These datasets can only be used for research purposes, and we request that they not be redistributed after obtaining access.

**Comment by rafalposwiata** [comment]
@KennethEnevoldsen @Muennighoff I will take this issue and add some datasets in the near future.

**Comment by KennethEnevoldsen** [comment]
Sounds great. It sounds like we can implement the private datasets as well using some sort of API key if we wish (so only people with access to the dataset can evaluate models, but the evaluation is publicly implemented)
